packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -322,4 +322,67 @@ export default class DedicatedServerDashboard {
       type: 'navigation',
     });
   }
+
+  onChangeIntervention(noIntervention) {
+    const { monitored } = this.server;
+    this.updatingNoIntervention = true;
+    this.Server.updateMonitoring(
+      this.$stateParams.productId,
+      monitored,
+      noIntervention,
+    )
+      .then(() => {
+        this.server.noIntervention = noIntervention;
```

Comment:
I'm a bit disturbed by this logic. `this.server.noIntervention` doesn't exist yet before updating monitoring ?

Reply:
its not required. I have removed it.